pkgs/os-specific/linux/kbd/default.nix

```
@@ -63,6 +63,7 @@ stdenv.mkDerivation rec {
   '';
 
   buildInputs = [ check pam ];
+  ${if stdenv.hostPlatform.isStatic then "NIX_LDFLAGS" else null}= [ "-laudit" ];
```

I think it would be _much_ more readable to write:

```suggestion
  NIX_LDFLAGS = lib.optional stdenv.hostPlatform.isStatic "-laudit";
```

It'd have to go to staging though, of course.  Do you need it urgently?

---

no. I copy pasted it from elsewhere.

---

done